src/azure-cli/requirements.py3.windows.txt

```diff
@@ -1,6 +1,6 @@
 adal==1.2.7
 antlr4-python3-runtime==4.7.2
-applicationinsights==0.11.7
+applicationinsights==0.11.9
```

Comment:
Why modify version other than recoveryservices?

Reply:
There is an unknown problem in the virtual environment, then change it back

Reply:
There are also occurrences of this issue in this PR. Please fix them, too.